src/prpy/util.py

```diff
@@ -2129,3 +2129,50 @@ def GetPointFrom(focus):
         raise ValueError('Focus of the point is an unknown object')
 
     return coord
+
+def concatenateTrajectories(traj_list):
```

Comment:
We follow OpenRAVE's naming convention by using `PascalCase` instead of `camelCase`.